index.html

```diff
@@ -0,0 +1,257 @@
+          </ul>
+          <button class="card-btn">See Project</button>
+        </div>
+      </div>
+    </section>
+    <!-- About Section -->
+    <section class="about-page" id="about-page">
+      <div class="about-container">
+        <div class="about-content">
+          <h2 class="about-title">About Myself</h2>
+          <p class="about-para">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+          <h3 class="about-contact">LET'S CONNECT</h3>
+          <ul class="social-icons">
+            <li class="icons">
+              <a href="#"><img src="images/Vector (15).png" alt="Twitter" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"
+                ><img src="images/Linkedin icon.png" alt="LinkedIn"
+              /></a>
+            </li>
+            <li class="icons">
+              <a href="#"><img src="images/Vector (11).png" alt="Medium" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"><img src="images/Vector (12).png" alt="GitHub" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"
+                ><img src="images/Vector (13).png" alt="AngelList"
+              /></a>
+            </li>
+          </ul>
+          <button class="about-btn">Get my resume</button>
+        </div>
+        <div class="about-lang">
+          <div class="languages">
+            <div class="lang-title">
+              <h3>Languages</h3>
+              <img  src="images/Union.svg" alt="arrow" />
+            </div>
+
+            <ul class="lang">
+              <li class="langs">
+                <img
+                 src="images/Ellipse 1.png"
+                  alt="JavaScript-Logo"
+                />
+                <h4 class="langs-text">JavaSript</h4>
+              </li>
+              <li class="langs">
+                <img src="images/Ellipse 2.png" alt="HTML5-Logo" />
+                <h4 class="langs-text">HTML</h4>
+              </li>
+              <li class="langs">
+                <img src="images/Ellipse 3.png" alt="CSS3-Logo" />
+                <h4 class="langs-text">CSS</h4>
+              </li>
+            </ul>
+          </div>
+          <div>
+            <ul>
+            <div class="framework">
+             
+             <li><h4 class="frame-title">Frameworks</h4></li>
+           <li><img src="images/Union2.svg" alt="arrow" /></li>   
+            </div>
+            <hr class="new1" />
+          </div>
+          <div>
+            <div class="skill">
+             <li><h4 class="skill-title">Skills</h4></li> 
+             <li><img  src="images/Union2.svg" alt="arrow" /></li>
+            </div>
+           <hr class="new1" />
+          </div>
```

Comment:
- -  You have done great so far. :+1: :+1: Your work and about section look great. But there are some changes you need to make before moving to the next milestone. :+1: 
- [ ] To better align and structure your code, The technology categories ("Languages", "Frameworks" and "Skills") needs to be in a ul, each of them in a li. I'll suggest you update this on your code. :smile: :wave:

Reply:
I have now placed them each inside an <ul> Please see the following script

<div class="languages">
            <div class="lang-title">
              <h3>Languages</h3>
              <img  src="images/Union.svg" alt="arrow" />
            </div>
            <ul>
              <li class="langs">
                <img src="images/Ellipse 1.png" alt="JavaScript-Logo" />
                <h4 class="langs-text">JavaSript</h4>
              </li>
            </ul>
            <ul>
              <li class="langs">
                <img src="images/Ellipse 2.png" alt="HTML5-Logo" />
                <h4 class="langs-text">HTML</h4>
              </li>
            </ul>
            <ul>
              <li class="langs">
                <img src="images/Ellipse 3.png" alt="CSS3-Logo" />
                <h4 class="langs-text">CSS</h4>
              </li>
            </ul>
          </div>
          <div>
            <ul>
              <div class="framework">
              <li><h4 class="frame-title">Frameworks</h4></li>
              <li><img src="images/Union2.svg" alt="arrow" /></li>   
              </div>
              <hr class="new1" />
            </ul>
          </div>
          <div>
            <ul>
              <div class="skill">
              <li><h4 class="skill-title">Skills</h4></li> 
              <li><img  src="images/Union2.svg" alt="arrow" /></li>
              </div>
            <hr class="new1" />
            </ul>
          </div>
        </div>